spec/index.bs

```diff
@@ -15308,24 +15324,37 @@ Eventually all of these should disappear but they are useful to avoid warning wh
 [=buffer internals/state/destroyed=]
 
 <script>
-// Small script to collect all xrefs that refer to different timelines and highlight
-// them with an appropriate style.
-
-const dataLinks = document.querySelectorAll("[data-link-type='abstract-op'], [data-link-type='dfn'], [data-link-type='idl'], [data-link-type='attribute']");
-for (const dataLink of dataLinks) {
-    const linkUrl = dataLink.getAttribute('href');
-    if (linkUrl && linkUrl.startsWith('#')) {
-      const dataLinkTarget = document.getElementById(linkUrl.substring(1));
-      if (dataLinkTarget) {
-          // Find the closest ancestor of the target (including the target itself)
-          // that contains a data-timeline.
-          const timelineElement = dataLinkTarget.closest('[data-timeline]');
-          if (timelineElement) {
-            // If we found a timeline, apply an appropriate style the link.
-            const timeline = timelineElement.getAttribute('data-timeline');
-            dataLink.setAttribute('data-timeline', timeline);
-          }
-      }
+window.addEventListener('load', () => {
```

Comment:
These needed to move into a window load event callback because it turns out the issue list is appended by bikeshed below this point, and inline scripts can only see HTML content added prior to the script block. Waiting for load ensures these JS fixups apply to the entire doc.